docs/GettingStarted.md

```diff
@@ -1,3 +1,269 @@
 # Getting Started
 
-This documentation isn't finished yet! In fact, this project is supposed to be very hush hush. Come back after we officially release!
+In this section, we will show you how to get started with Shasta.
+
+Shasta wraps the Redux's store and allows you to write simpler
+reducers and actions. Under the hood, it uses [redux-actions](https://github.com/acdlite/redux-actions)
+and [immutablejs](https://facebook.github.io/immutable-js/).
+
+Shasta also supports the [redux-devtools Chrome extension](https://github.com/zalmoxisus/redux-devtools-extension)
+out-of-the-box. This plugin will help you track the state of your application.
+
+## Installation and setup
+
+We need babel to compile our ES2015 sources.
```

Comment:
i think this guide might be overcomplicated, a shasta getting started should show a very basic way to use shasta not how to scaffold a whole project using babel. the getting started should be simple enough to run in a codepen.